tests/run/worktree/data/prepare/plan.fmf

```diff
@@ -1,6 +1,6 @@
 provision:
     how: container
 prepare:
-    script: echo content > file
+    script: echo content > $TMT_PLAN_DATA/file
 execute:
-    script: grep content file
+    script: grep content $TMT_PLAN_DATA/file
```

Comment:
I'm not sure about this change. Preparing a file during the `prepare` step and then using it during `execute` without need to `pull()` it from the guest is a valid use case. I'm afraid this test failing with the `virtual` provision is just confirming #964.

Reply:
OK. So we can keep test as is to check for #964